testnets/kimanetworktestnet/chain.json

```diff
@@ -25,61 +25,90 @@
       }
     ]
   },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uKIMA"
+      }
+    ],
+    "lock_duration": {
+      "time": "259200s"
+    }
+  },
   "codebase": {
-    "git_repo": "https://github.com/kima-finance/kima-blockchain",
-    "recommended_version": "v0.4.3",
+    "git_repo": "https://github.com/kima-finance/kima-blockchain/",
+    "recommended_version": "v0.4.6",
     "compatible_versions": [
-      "v0.4.3"
+      "v0.4.6"
     ],
     "genesis": {
-      "genesis_url": "http://data-testnet.kima.finance/genesis.json"
+      "genesis_url": "http://archive.sardis.kima.network/genesis.json"
+    },
+    "consensus": {
```

Comment:
Please copy or move these properties (consensus, lenguage, sdk, ibc) into the version object in the "versions.json" file.